Ginger/Ginger/UserControlsLib/UCTreeView/SingleItemTreeViewSelectionPage.xaml.cs

```diff
@@ -128,7 +128,9 @@ public List<object> ShowAsWindow(string windowTitle="", eWindowShowStyle windowS
 
             if (windowTitle == string.Empty)            
                 windowTitle = mitemTypeName + " Selection";
-            GenericWindow.LoadGenericWindow(ref mPageGenericWin, App.MainWindow, windowStyle, windowTitle, this, winButtons, true, "Close", CloseWinClicked, startupLocationWithOffset: startupLocationWithOffset);
+
+            GenericWindow.LoadGenericWindow(ref mPageGenericWin, App.MainWindow, windowStyle, windowTitle, this, winButtons, true, "Close", CloseWinClicked, 
```

Comment:
i don't think adding this mainWindowParent boolean is needed, if you found that the issue is due to the GenericWindows Owner page that mean that what causing the problem is that we do not send the correct Owner page to it, and controling the Owner is already supported, you can see in this GenericWindow.LoadGenericWindow() call we always sending the App.MainWindow as the owner and that probably what causing the issue, i belive that if you will add optional parameter to ShowAsWindow(Page parentPage = null) to get the parent page and then you will pass it to GenericWindow.LoadGenericWindow() instead of App.MainWindow (send App.MainWindow in case parentPage is null) then the Owner will be set correctly and problem will be solved.
when you call ShowAsWindow from your wizard, send the parentPage as the Wizard Window so it won't point to MainWindow

Reply:
I already tried the same sending the "parrentPage" as parameter to the function but the "Owner" requires the "System.Window" type, and we have wizard as the "Page" which cannot be assigned to the "Owner" property, it gives exception in that case, so let me know if i should rollback the changes done?